hail_scripts/v02/utils/computed_fields/vep.py

```diff
@@ -125,7 +125,20 @@ def get_expr_for_vep_sorted_transcript_consequences_array(vep_root, include_codi
     ]
 
     if include_coding_annotations:
-        selected_annotations.extend(["amino_acids", "lof", "lof_filter", "lof_flags", "lof_info", "protein_id"])
+        selected_annotations.extend(
+            [
```

Comment:
are you sure you want both pred. and score fields? seqr currently only shows the predictions:

![image](https://user-images.githubusercontent.com/6240170/45121115-24465080-b12e-11e8-9cab-c81cb8ea581d.png)

Reply:
You're right, it is only the prediction fields that are needed.

Reply:
The prediction values are also a little weird. IIRC they have ;-separated values that are repeated multiple times (for different proteins off of the same transcript). 
You might want to pre-process them to a single value per transcript.

Reply:
I checked the predictions on ClinVar variants and didn't see anything that looked like it had multiple values concatenated into one string.